server/config/forms/incidentDateValidation.ts

```diff
@@ -29,7 +30,7 @@ export function dateValidation(date: string, helpers: joi.CustomHelpers) {
 export function timeValidation(time: string, helpers: joi.CustomHelpers) {
   const fullDate = helpers.state.ancestors[0].value
 
-  if (fullDate && moment(fullDate).isBetween(moment.now(), moment().endOf('day'), 'minutes')) {
+  if (fullDate && moment(fullDate).isBetween(moment.now(), moment().endOf('day'), 'seconds')) {
```

Comment:
Is this needed? I don't see a test that corresponds to this change and why it would be necessary?

Reply:
So, minutes was allowing the user to input 23.59 form the current day and i was being accepted. Using seconds prevents that